src/services/mentees.js

```diff
@@ -1150,7 +1207,7 @@ module.exports = class MenteesHelper {
 	 * @param {String} userId - User ID of the mentee.
 	 * @returns {Promise<Object>} - Mentee extension details.
 	 */
-	static async getMenteeExtension(userId, organizationId, tenantCode) {
+	static async getMenteeExtension(userId, organizationCode, tenantCode) {
```

Comment:
@sumanvpacewisdom  in getMenteeExtension() we should use cache,  try using cache all the places, if the same required data is present cache